druid-shell/src/platform/windows/window.rs

```diff
@@ -378,8 +381,14 @@ impl WndProc for MyWndProc {
                         self.handle.borrow().rect_to_px(rect),
                     );
                     if let Some(ref mut ds) = s.dcomp_state {
+                        let params = DXGI_PRESENT_PARAMETERS {
+                            DirtyRectsCount: 1,
+                            pDirtyRects: &mut rect as *mut RECT,
+                            pScrollRect: null_mut(),
+                            pScrollOffset: null_mut(),
+                        };
                         if !ds.sizing {
-                            (*ds.swap_chain).Present(1, 0);
+                            (*ds.swap_chain).Present1(1, 0, &params as *const _);
```

Comment:
Let's not hide the type with `_`. Can we do `*const DXGI_PRESENT_PARAMETERS`?

Reply:
Why not just `(*ds.swap_chain).Present1(1, 0, &params);`? Seems to work for me. I personally consider `as *const _` to be idiomatically, as it reads "make this into the type winapi expects". But it's far nicer if the type just works without casting.